docs/pages/management/operations/db-ca-migrations.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Database CA Migrations
+description: How to complete Teleport Database CA migrations.
+---
+
+In Teleport, self-hosted databases must be configured with certificates to
+enable mTLS authentication via the Teleport Database Service.
+
+Teleport 15 introduced a new `db_client` certificate authority (CA) to split the
```

Comment:
Do we want to mention the patch versions of the older releases?

Reply:
Yes good point, I should have mentioned what I was planning for that.

The db ca split backports have both been merged but have not been in a release yet.
I have a TODO note to update the versions documented here when we have the v13/v14 patch versions and then backport that - probably sometime next week.